SSA/Projects/RISCV64/Semantics.lean

```diff
@@ -1,7 +1,6 @@
 import SSA.Core.MLIRSyntax.EDSL
 import SSA.Projects.InstCombine.ForLean
 import SSA.Projects.RISCV64.Tactic.SimpRiscVAttr
-set_option maxHeartbeats 1000000000000000000
```

Comment:
This change seems to be unrelated. Let me pull it out to help with making this upstreaming quick yet principled.

Reply:
This has been upstreamed in https://github.com/opencompl/lean-mlir/pull/1547. I will update this PR to latest main to make the diff smaller.